templates/careers/index.html

```diff
@@ -7,149 +7,94 @@
+      <p>Join an intense global mission &ndash; to deliver the world's best open source experience,<br class="u-hide--small">from platform to application</p>
     </div>
-    <div class="row u-equal-height">
-      <div class="col-6 p-card--strip-thin">
-        <h2 class="p-card--strip-thin__title">Engineering</h2>
-        <p class="p-card--strip-thin__content">Open source is transforming the entire stack. This is your chance to be right at the center of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud to high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
-        <a href="/careers/engineering">More&nbsp;&rsaquo;</a>
-      </div>
-      <div class="col-6 p-card--strip-thin">
-        <h2 class="p-card--strip-thin__title">TechOps</h2>
-        <p class="p-card--strip-thin__content">Getting it done right is the best feeling in the world. Whether it’s running complex technical infrastructure or challenging projects, your consistent focus on data, organisation and teamwork means you’re a force for good in the battle against entropy. Time zones are easy and technical customers are your favourite.</p>
-        <a href="/careers/techops">More&nbsp;&rsaquo;</a>
-      </div>
-    </div>
-
+  </div>
+</section>
 
-    <div class="row u-equal-height">
-      <div class="col-4 p-card--strip-thin">
-        <h4 class="p-card--strip-thin__title">Finance</h4>
-        <p class="p-card--strip-thin__content">Finance at Canonical isn’t just about numbers. It’s about connecting with people across the business, helping them to understand the value they bring, as well as promoting the commercial focus that will grow our brand.</p>
-        <a href="/careers/finance">More&nbsp;&rsaquo;</a>
-      </div>
-      <div class="col-4 p-card--strip-thin">
-        <h4 class="p-card--strip-thin__title">Web &amp; design</h4>
-        <p class="p-card--strip-thin__content">The design team at Canonical crafts and builds the user-experience for sophisticated technical specialists across a wide range of industries and geographies. We design and build websites, complex web apps and support Ubuntu itself.</p>
-        <a href="/careers/web-and-design">More&nbsp;&rsaquo;</a>
-      </div>
-      <div class="col-4 p-card--strip-thin">
-        <h4 class="p-card--strip-thin__title">Marketing</h4>
-        <p class="p-card--strip-thin__content">We deliver the right technology message to a highly competent audience, at exactly the right moment. Success depends on having good instincts for tech trends and topics and a natural inclination to leadership.</p>
-        <a href="/careers/marketing">More&nbsp;&rsaquo;</a>
-      </div>
+<section class="p-strip--light is-deep u-no-padding--bottom">
```

Comment:
Is `u-no-padding--bottom` intentionally added here?
maybe This got already design +1 right? I'm just leaving a comment as I see a bit of padding differences here compared to figma. Feel free to disregard this comment :)  
<img width="835" alt="Screenshot 2022-10-05 at 2 10 21 pm" src="https://user-images.githubusercontent.com/57550290/194057560-6b82a35f-6cdf-49c0-980f-cf2deedbf172.png">